docs/content/guides/author-apps/azure/howto-azure-resources/index.md

```diff
@@ -32,27 +32,33 @@ Begin by running [`rad init --full`]({{< ref rad_init >}}). Make sure to configu
 rad init --full
 ```
 
-## Step 2: Define a Radius Environment
+## Step 2: Create a `bicepconfig.json` in your application's directory 
+
+{{< read file= "/shared-content/installation/bicepconfig/manual.md" >}}
+
+More information on how to setup a `bicepconfig.json` can be found [here]({{< ref "/guides/tooling/bicepconfig/overview" >}})
+
+## Step 3: Define a Radius Environment
```

Comment:
As discussed lets add a note that says select "no " to setup application in the previous  step so that the steps remain the same.

Reply:
Updated